modules/tinymce/src/plugins/autolink/test/ts/browser/AutoLinkPluginTest.ts

```diff
@@ -27,34 +35,32 @@ UnitTest.asynctest('browser.tinymce.plugins.autolink.AutoLinkPluginTest', (succe
     editor.setContent('<p>' + modifiedurl + '</p>');
     LegacyUnit.setSelection(editor, 'p', modifiedurl.length);
     KeyUtils.type(editor, ')');
-    Assert.eq('Create a link of an eclipsed url', `<p>(<a href="${expectedUrl || url}">${url + dot}</a>)</p>`, editor.getContent());
+    assert.equal(editor.getContent(), `<p>(<a href="${expectedUrl || url}">${url + dot}</a>)</p>`, 'Create a link of an eclipsed url');
   };
 
   const typeNewlineURL = (editor: Editor, url: string, expectedUrl?: string, withDotAtTheEnd?: boolean): void => {
     const dot = withDotAtTheEnd ? '.' : '';
     editor.setContent('<p>' + url + dot + '</p>');
     LegacyUnit.setSelection(editor, 'p', url.length);
     KeyUtils.type(editor, '\n');
-    Assert.eq('Create link with newline', `<p><a href="${expectedUrl || url}">${url}</a></p><p>${withDotAtTheEnd ? '.' : '&nbsp;'}</p>`, editor.getContent());
+    assert.equal(editor.getContent(), `<p><a href="${expectedUrl || url}">${url}</a></p><p>${withDotAtTheEnd ? '.' : '&nbsp;'}</p>`, 'Create link with newline');
   };
 
   const assertNoLink = (editor: Editor, input: string, text?: string): void => {
-    Assert.eq('Should not convert to link', `<p>${text || input}&nbsp;</p>`, typeUrl(editor, input));
+    assert.equal(typeUrl(editor, input), `<p>${text || input}&nbsp;</p>`, 'Should not convert to link');
   };
 
   const assertIsLink = (editor: Editor, input: string, link: string, withDotAtTheEnd?: boolean, text?: string): void => {
     const dot = withDotAtTheEnd ? '.' : '';
-    Assert.eq('Should be convert to link', `<p><a href="${link}">${text || input}</a>${dot}&nbsp;</p>`, typeUrl(editor, (input + dot)));
+    assert.equal(typeUrl(editor, (input + dot)), `<p><a href="${link}">${text || input}</a>${dot}&nbsp;</p>`, 'Should be convert to link');
   };
 
-  const test = (label: string, runTest: (editor: Editor) => void): void => {
-    suite.test(label, (editor) => {
-      editor.focus();
-      runTest(editor);
-    });
-  };
+  beforeEach(() => {
+    hook.editor().focus();
```

Comment:
I don't believe any of these tests require `editor.focus()` to run correctly

Reply:
Removed and replaced with a focus on init instead